src/components/AccountsManagementComponent.jsx

```diff
@@ -159,140 +190,341 @@ class AccountsManagementComponent extends Component {
+        <TableBody>
+          {categorizedAccounts.map((account, i) => {
+            const expanded = expandedRow[account.id]
+            const rowCellClassName = expanded
+              ? clsx(classes.cellOverFlow, classes.noBottomBrd)
+              : classes.cellOverFlow
+            return (
+              <React.Fragment key={'wide' + i}>
+                <TableRow
+                  key={i}
+                  onClick={() => {
+                    this.expandRow(account)
+                  }}
+                  hover
+                  selected={expanded}
+                  style={{ cursor: 'pointer' }}
+                >
+                  <TableCell
+                    align='left'
+                    className={clsx(rowCellClassName, classes.avatar)}
+                    padding='none'
+                  >
+                    <Avatar
+                      style={{ borderRadius: '0px', marginLeft: 15 }}
+                      src={getWalletLogo(account.walletType)}
+                    ></Avatar>
+                  </TableCell>
+
+                  <TableCell align='left' className={rowCellClassName}>
+                    <Box>
+                      <Typography variant='body2'>{account.name}</Typography>
+                      <Typography variant='caption'>
+                        {getWalletTitle(account.walletType)}, {getCryptoTitle(account.platformType)}
+                      </Typography>
+                    </Box>
+                  </TableCell>
+
+                  <TableCell align='left' className={rowCellClassName}>
+                    {account.status === accountStatus.syncing ? (
+                      <Skeleton style={{ margin: '0px', width: '100%', minWidth: '100px' }} />
+                    ) : account.assets.length === 1 ? (
+                      `${getCryptoSymbol(
+                        account.assets[0].cryptoType
+                      )} ${this.toCurrencyLocaleString(account.assets[0].balanceInStandardUnit)}`
+                    ) : (
+                      'Multiple Coins'
+                    )}
+                  </TableCell>
+                  <TableCell align='left' className={rowCellClassName}>
+                    $ {account.totalMarketValue}
+                  </TableCell>
+                  <TableCell align='left' className={rowCellClassName}>
+                    {expanded ? (
+                      <ExpandLessIcon className={classes.iconBtn} />
+                    ) : (
+                      <ExpandMoreIcon className={classes.iconBtn} />
+                    )}
+                  </TableCell>
+                </TableRow>
+                {expanded &&
+                  account.assets.map((asset, i) => {
+                    const last = account.assets.length - 1 === i
+                    return (
+                      <TableRow>
+                        <TableCell
+                          padding='none'
+                          className={last ? undefined : classes.noBottomBrd}
```

Comment:
is it possible to do the following?
```suggestion
                          className={!last && classes.noBottomBrd}
```

Reply:
`className={!last && classes.noBottomBrd}` would cause 'className cannot be boolean' error.